### PR TITLE
Feature/header component

### PR DESCRIPTION
**Pull request by acis**
Improvements for #1 
- Fixes for SVGs for the header component.

- Made bottom header optional

### CODE REVIEW

**Comment by aniri** [comment]
Hey @acis ! some fresh updates were made to the header menu items, could yopu please update them to the following (replace menuitem1 and 2 with):
    Despre (link to About)
    Ecosistemul Covid-19 (link to code4.ro/ro/apps)
    Sprijină proiectul (link to code4.ro/ro/doneaza)

File: src/components/header/header.js [hunk 1]
```diff
@@ -15,22 +21,28 @@ export const Header = ({ name, Logo, MenuItems, ProfileItems }) => {
...
+            <span>În parteneriat cu </span>
+            <a
+              href="https://www.gov.ro/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <GovroSvg />
+            </a>
+          </div>
         </div>
```
Comment (moonflare) [inline src/components/header/header.js]: Hey @acis There is a separate issue for this part. We already have a PR for it: https://github.com/code4romania/taskforce-fe-components/pull/33

File: src/components/header/header.js [hunk 2]
```diff
@@ -39,5 +51,6 @@ Header.propTypes = {
   name: PropTypes.string.isRequired,
```
Comment (moonflare) [inline src/components/header/header.js]: This prop should be optional and be conditioned. In the design, I can see that we only have the logo. So I think we can show it only if we pass it down as a prop. Something like this on `L18`

```js
{name && <div className="title">{name}</div>}
```

File: src/components/search-input/search-input.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ export const SearchInput = ({
     if (value) {
       setCurrentValue(value);
     }
-  }, []);
+  }, [value]);
```
Comment (moonflare) [inline src/components/search-input/search-input.js]: The reason it was set to only update the `currentValue` from outside the component once is that we don't want to rely on a state from outside the component. So if we want to initialize the `currentValue`, we can, but any other change that comes after we rely on the internal state and then do the callback to notify parent components. Please let me know if this makes sense.
```suggestion
  }, []);
```

**Comment by moonflare** [comment]
> Hey @acis ! some fresh updates were made to the header menu items, could you please update them to the following (replace menuitem1 and 2 with):
> Despre (link to About)
> Ecosistemul Covid-19 (link to code4.ro/ro/apps)
> Sprijină proiectul (link to code4.ro/ro/doneaza)

@aniri There is no need for changes. The component doesn't know about what links we provide to it. ` menuitem1 and 2` are just examples in Storybook. When we use the component in the applications, we just give it the right links you mentioned above 🙌

**Comment by aniri** [comment]
> > Hey @acis ! some fresh updates were made to the header menu items, could you please update them to the following (replace menuitem1 and 2 with):
> > Despre (link to About)
> > Ecosistemul Covid-19 (link to code4.ro/ro/apps)
> > Sprijină proiectul (link to code4.ro/ro/doneaza)
> 
> @aniri There is no need for changes. The component doesn't know about what links we provide to it. ` menuitem1 and 2` are just examples in Storybook. When we use the component in the applications, we just give it the right links you mentioned above

ok, got it! thanks for clarifying this :smile:

File: src/components/header/header.js [hunk 4]
```diff
@@ -1,42 +1,24 @@
...
-import GovLogoSVG from "../../icons/gov_logo.svg";
-import C4RLogoSvg from "../../icons/c4r_logo.svg";
 
 export const Header = ({ name, Logo, MenuItems, ProfileItems }) => {
   return (
     <header className="App-header">
       <div className="top-header">
         <div className="app-logo">{Logo}</div>
-        <div className="title">{name}</div>
+        <div className="title">{name ? name : ""}</div>
```
Comment (surdu) [inline src/components/header/header.js]:
```suggestion
        <div className="title">{ name || ""}</div>
```

Comment (surdu) [reply src/components/header/header.js]: Other than this small change, everything looks awesome!

**Pull request merged**